client/src/components/Home/index.tsx

```diff
@@ -213,6 +257,44 @@ export default class Home extends React.Component<any, IHomeState> {
     return (
       <div>
         <h1>Bot Analytics</h1>
+        
+      <Button
```

Comment:
Hi, So currently the button is indeed just below the 'bot analytics' headline:
![ButtonLocation](https://user-images.githubusercontent.com/13205761/27768165-f31b7eb0-5f15-11e7-88de-98a132d2e8e9.PNG)

However, in this page (the home component) there is no 'sub toolbar', so I was thinking about aligning the button the right, it will give the same affect. thoughts?

no sub-toolbar:
![no sub toolbar](https://user-images.githubusercontent.com/13205761/27768211-82838ef8-5f16-11e7-9226-819a015fc7a8.PNG)